docs/source/tutorials/basic_renewal_model.qmd

```diff
@@ -305,7 +332,7 @@ axes.plot(x_data, mean_Rt[0], color="C0", label="Mean")
 axes.legend()
 axes.set_title("Posterior Effective Reproduction Number", fontsize=10)
 axes.set_xlabel("Time", fontsize=10)
-axes.set_ylabel("$R_t$", fontsize=10)
+axes.set_ylabel("$\mathcal{R}(t)$", fontsize=10)
```

Comment:
This doesn't look right
![image](https://github.com/user-attachments/assets/e486bb16-266c-4e0d-9aec-e0dc3f1ff2e0)

Reply:
I see above you are trying to ignore `nan` padding, but are we sure there are `nan`s? It looks like we can compute the mean.

Reply:
Fixed in 0afa30e

Explicitly removed the `nan` values when plotting, rather than relying on internal removal.

In the process, I caught an issue with how the posterior means were being computed. It was computing posterior means chain-by-chain rather than across all chains, but then only using the first chain's means. It's easier to see (and avoid) the potential for this error when using built-in `InferenceData.mean()` or `.median()` methods (as opposed to `np.mean` or `np.median`). I have implemented this change.